ideascube/forms.py

```diff
@@ -69,11 +69,24 @@ def save(self, *args, **kwargs):
 class UserImportForm(forms.Form):
     source = forms.FileField(required=True)
 
+    def canonicalize_row(self, row):
+        result = {}
+        data_fields = User.get_data_fields()
+        data_fields = {str(f.verbose_name): f.name for f in data_fields}
+
+        for field in row:
+            # TODO: What to do on KeyError?
```

Comment:
The KeyError could also be raised in case of a translation change.
If we update the field translation between the export and the import, names will not match with the field.

I'm the feeling that what is wrong is the header translation at the export itself.

@barbayellow what is the purpose of the export ?
Are we exporting to backup/exchange with another software or are we exporting for a human reading/editing ?